src/Files.App/App.xaml.cs

```diff
@@ -369,6 +383,105 @@ public static void SaveSessionTabs()
 				}
 			})
 			.ToList();
+			RemoveThisInstanceTabs();
+		}
+
+
+
+		public static MemoryMappedFile CheckSharedMemory()
```

Comment:
Can you move those static mothods into Helpers/AppLifecycle.cs you are going to create so we wont see any mess?